frontend/src/components/AccountReceiveTable.vue

```diff
@@ -348,21 +347,21 @@ function useAdvancedFeatures(spendingKeyPair: KeyPair) {
 
   // Generate string that explains scan settings that were used
   const scanDescriptionString = computed(() => {
-    const suffix = scanPrivateKey.value ? vm.$i18n.t('AccountReceiveTable.custom-prv-key') : '';
+    const suffix = scanPrivateKey.value ? vm.$i18n.tc('AccountReceiveTable.custom-prv-key') : '';
     const hasStartBlock = Number(startBlock.value) >= 0;
     const hasEndBlock = Number(endBlock.value) >= 0;
-    let msg = `${vm.$i18n.t('AccountReceiveTable.scanned-from-block').toString()} ${Number(
-      startBlock.value
-    )} ${vm.$i18n.t('AccountReceiveTable.to').toString()} ${Number(endBlock.value)}`; // default message
+    let msg = `${vm.$i18n.tc('AccountReceiveTable.scanned-from-block')} ${Number(startBlock.value)} ${vm.$i18n.tc(
```

Comment:
Just curious, do these need to be cast to a number, or can we remove that cast to clean it up a bit. Wondering if we need to cast because the linter complains?

Reply:
Sorry, no I mean can we make this change to make it a bit cleaner/easier to read?

```suggestion
    let msg = `${vm.$i18n.tc('AccountReceiveTable.scanned-from-block')} ${startBlock.value} ${vm.$i18n.tc(
```

Reply:
They originally had the Number casting. After removing them you get `Invalid type "number | undefined" of template literal expression.` lint error when running `yarn build`. So I kept them in.